api/v1beta1/azuremanagedcontrolplane_types.go

```diff
@@ -213,6 +213,10 @@ type AzureManagedControlPlaneSpec struct {
 	// OIDCIssuerProfile is the OIDC issuer profile of the Managed Cluster.
 	// +optional
 	OIDCIssuerProfile *OIDCIssuerProfile `json:"oidcIssuerProfile,omitempty"`
+
+	// DisableLocalAccounts - If set to true, getting static credential will be disabled for this cluster. Expected to only be used for AAD clusters.
```

Comment:
```suggestion
	// DisableLocalAccounts disables getting static credentials for this cluster when set. Expected to only be used for AAD clusters.
```

Reply:
I copied the comments from the azure sdk :) 

https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/containerservice/armcontainerservice/v4@v4.3.0#ManagedClusterProperties.DisableLocalAccounts